rust/operator-binary/src/druid_controller.rs

```diff
@@ -617,11 +651,15 @@ fn build_rolegroup_statefulset(
     let mut pb = PodBuilder::new();
     pb.node_selector_opt(druid.node_selector(rolegroup_ref));
 
+    let (ldap_auth_mounts, ldap_auth_cmd) =
+        get_ldap_secret_volume_and_volume_mounts_and_commands(ldap_settings);
+
```

Comment:
Hey! I see you have written your own function to create and mount volumes. For the sake of consistency, could you call the framework functions instead? [this PR](https://github.com/stackabletech/issues/issues/322) links to the PR with the Framework changes, as well as a bunch of other example PRs of how to use the Framework functions.

Reply:
Yeah i agree with felix here. And this wrapper should be implemented on `DruidLdapSettings`.

Reply:
Switching worked well 👌